File: 백준/Bronze/1978. 소수 찾기/소수 찾기.js
let input = require("fs")
  .readFileSync("/dev/stdin")
  .toString()
  .trim()
  .split("\n");

let N = parseInt(input[0]);

let result = 0;
let nums = input[1].split(" ").map(Number);

for (let i = 0; i < N; i++) {
  if (nums[i] === 1) {
    continue;
  } else {
    let count = 0;
    for (let j = 2; j < nums[i]; j++) {
      if (nums[i] % j === 0) {
        count += 1;
      }
    }

    if (count === 0) {
      result += 1;
    }
  }
}

console.log(result);

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.kt
fun main() {
    val N = readln().toInt()
    val nums = readln().split(" ").map { it.toInt() }
    var count = 0

    for (i in 0 until N) {
        if (nums[i] == 1) continue
        var isPrime = true
        for (j in 2 until nums[i]) {
            if (nums[i] % j == 0) {
                isPrime = false
                break
            }
        }
        if (isPrime) count++
    }
    println(count)
}